packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -18,7 +18,7 @@ import {
 import SyntheticEvent from 'events/SyntheticEvent';
 import invariant from 'fbjs/lib/invariant';
 
-import BrowserEventConstants from '../events/BrowserEventConstants';
+import {topLevelTypes} from '../events/BrowserEventConstants';
```

Comment:
Oops I think this might not be enough now? We need to do the same for `mediaEventTypes` or we'll "lose" some events?

Reply:
Ah man. I think you're right. Darn.